src/components/onboarding_comp.js

```diff
@@ -0,0 +1,26 @@
+function Onboarding() {
+  return (
+    <div className="space-y-6">
+      <div className="text-[#6D6AEC] text-[43px] font-bold">
+        Get Started With Carai
+      </div>
+      <ul className="marker:text-[#6D6AEC] list-disc px-6 space-y-6 font-bold text-lg">
+        <li>Share, Review & Improve your Code.</li>
+        <li>Connect with peers and code together.</li>
+        <li>Communicate with peers and code together.</li>
+        <li>Keep track of changes with code versionong.</li>
+        <li>Stop coding alone and start collaborating today.</li>
+      </ul>
+      <div className="flex flex-col space-y-6 ">
```

Comment:
This should be `space-y-3` not `space-y-6`

Reply:
Noted